util/LLgen/src/LLgen.c

```diff
@@ -2,9 +2,6 @@
 #include "Lpars.h"
```

Comment:
Just checking: this entire file is autogenerated by LLgen and you didn't edit it by hand, right?

Reply:
Yes, you are correct. I did not touch this part.